enforce/enforce.go

```diff
@@ -26,9 +27,19 @@ func ENFORCE(query interface{}, args ...interface{}) {
 				panic(t)
 			}
 		}
+	case nil:
```

Comment:
Good catch on my incorrect usage :)
Lets do it as follows instead of adding a different function call; the default behaviour for calling enforce should be failure.

```suggestion
	case string:
		{
			log.Println("ENFORCE:", query.(string), args)
			panic(t)
		}
	default:
		log.Println("ENFORCE: incorrect usage of enforce with type: ", fmt.Sprintf("%T", t), "-", t, "-", args)
		panic(t)
```